src/main/java/org/fundacion/jala/converter/models/FileSQL.java

```diff
@@ -13,13 +13,17 @@
 import javax.persistence.EntityManagerFactory;
 import javax.persistence.Persistence;
 
+/**
```

Comment:
Add last name of mother to Juan Pablo Gonzalez Alvarado

Reply:
ok thanks gus